module/account/src/Infrastructure/Persistence/Query/DbalPrivilegeQuery.php

```diff
@@ -46,6 +49,32 @@ public function getPrivileges(bool $hidden = false): array
         return $qb->execute()->fetchAll();
     }
 
+    public function getPrivilegesEndPoint(): array
+    {
+        $qb = $this->connection->createQueryBuilder()
+            ->select(['id', 'name'])
+            ->from(self::PRIVILEGES);
+
+        return $qb->execute()->fetchAll();
+    }
+
+    /**
+     * @param Privilege[] $privileges
+     */
+    public function getPrivilegesEndPointByBusiness(array $privileges): array
```

Comment:
i think getPrivilegesEndPointByPrivilegesGroup will be better

Reply:
and endpoint is one word